src/Features/Core/Portable/xlf/FeaturesResources.cs.xlf

```diff
@@ -247,7 +247,7 @@ Ujistěte se, že specifikátor tt použijete pro jazyky, pro které je nezbytn
       </trans-unit>
       <trans-unit id="Convert_to_record">
         <source>Convert to record</source>
-        <target state="new">Convert to record</target>
+        <target state="translated">Převést na záznam</target>
```

Comment:
Feels like record shouldn't be translated?
The original resources seems to be a Locked comment and/or single quotes around "record"

Reply:
Please indicate as an xlf comment if the word 'record' is locked or not.